api-schema.graphql

```diff
@@ -992,4 +992,4 @@ input VerifyIdentityChallengeInput {
   provider: IdentityProvider!
   providerId: String!
   signature: String!
-}
\ No newline at end of file
+}
```

Comment:
Somehow on your PR the newline gets removed. It might be that the pre-commit hooks in the repo are not running properly.